payment/plugin.js

```diff
@@ -0,0 +1,102 @@
+/*
+* Razorpay Payment Button Plugin
+* @author Razorpay <neeraj.s@razorpay.com>
+* @version 1.0
+*/
+(function ($) {
+	
+	CKEDITOR.plugins.add('payment', {
+		lang: [ 'en'],
+		init: function (editor) {
+			editor.addCommand('payment', new CKEDITOR.dialogCommand('payment', {
+			 allowedContent: true,
+			}));
+			
+			editor.ui.addButton('PaymentButton', {
+				label : editor.lang.payment.button,
+				toolbar : 'insert',
+				command : 'payment',
+				icon : this.path + 'images/icon.png'
+			});
+
+			CKEDITOR.dialog.add('payment', function (instance) {
+				var video,
+					disabled = editor.config.payment_disabled_fields || [];
+
+				return {
+					title : editor.lang.payment.title,
+					minWidth : 510,
+					minHeight : 200,
+					onShow: function () {
+						for (var i = 0; i < disabled.length; i++) {
+							this.getContentElement('paymentButtonPlugin', disabled[i]).disable();
+						}
+					},
+					contents :
+						[{
+							id : 'paymentButtonPlugin',
+							expand : true,
+							elements :
+								[
+								{
+									type: 'select',
+									id: 'paymentbutton',
+									label: 'Select Payment Button',
+									items: this.getPaymentButtons(),
+									'default': '',
+									onChange: function( api ) {
+										
+										// this = CKEDITOR.ui.dialog.select
+										
+									}
+								},
+								
+							]
+						}
+					],
+					onOk: function()
+					{
+						var content = '';
+						var paymentId ='';
+						/**
+						 * Initialize razorpay buttons from api
+						**/
+						if (this.getContentElement('paymentButtonPlugin', 'paymentbutton').isEnabled()) 
+						{
+							paymentId = this.getValueOf('paymentButtonPlugin', 'paymentbutton');
+							content += '<div>Use Below Button to pay<form><script src="https://checkout.razorpay.com/v1/payment-button.js" data-payment_button_id=' + paymentId +' async> </script> </form></div>';
+						
+						}
+						else {
+						}
+
+						var element = CKEDITOR.dom.element.createFromHtml(content);
+						var instance = this.getParentEditor();
+						instance.insertElement(element);
+					}
+				};
+			});
+		}
+	});
+})(jQuery);
+
+
+
+function getPaymentButtons(){
```

Comment:
Kindly check the consistency for function nomenclature camelcase or snake case

Reply:
changed function in snake case